crates/turbopack-dev-server/src/source/mod.rs

```diff
@@ -246,29 +266,29 @@ pub struct ContentSourceData {
     pub cache_buster: u64,
 }
 
-type Chunk = Result<Bytes, BodyError>;
+pub type BodyChunk = Result<Bytes, BodyError>;
```

Comment:
```suggestion
pub type BodyChunk = Result<Bytes, SharedError>;
```

BodyError coverts everything to a string loosing all the structure. We want to avoid that.

Reply:
This requires a slightly larger change, and it's complicated by `ShareError` not being serializable. That would prevent any body response from being cachable, which I think is bad for our end goal? Anyways, I'd like to defer this to a follow up PR for the time being.

Reply:
> That would prevent any body response from being cachable, which I think is bad for our end goal?

It would still be cacheable in memory.

While I like to have static rendering to be cacheable, next.js doesn't cache rendering at all in dev and always server-renders a fresh page.